doc/conf.py

```diff
@@ -35,7 +35,7 @@
 copyright = "2021, Trevor Manz"
 author = "Trevor Manz"
 
-version = get_version(root="..", relative_to=__file__)
+version = gosling.__version__
```

Comment:
Running `hatch run docs` gives an error here saying that `gosling.__version__` does not exist.

I don't know if this is relevant, but would it be because I don't have `gosling/_version.py` on my local machine since it is included in `.gitignore`? I am just guessing from the name of the file..

Reply:
Never mind. I think my local code was somehow behind this branch. It works well.

Reply:
great!